# CvL: Ran into this issue https://bugreports.qt.io/browse/QTBUG-11923?focusedCommentId=233276&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-233276
# Comments in the header seem to suggest the macro expansion should fix it, but it does not
# The error we get is a missing pugixml.cpp
# Since pugixml by default is not configured to use header-only mode, we might as well comment it
diff -Nru pugixml-1.11.4-orig/src/pugixml.hpp pugixml-1.11.4/src/pugixml.hpp
--- pugixml-1.11.4-orig/src/pugixml.hpp	2021-06-24 10:24:25.117459425 +0200
+++ pugixml-1.11.4/src/pugixml.hpp	2021-06-24 10:25:21.338514908 +0200
@@ -1468,10 +1468,10 @@
 
 // Make sure implementation is included in header-only mode
 // Use macro expansion in #include to work around QMake (QTBUG-11923)
-#if defined(PUGIXML_HEADER_ONLY) && !defined(PUGIXML_SOURCE)
-#	define PUGIXML_SOURCE "pugixml.cpp"
-#	include PUGIXML_SOURCE
-#endif
+//#if defined(PUGIXML_HEADER_ONLY) && !defined(PUGIXML_SOURCE)
+//#	define PUGIXML_SOURCE "pugixml.cpp"
+//#	include PUGIXML_SOURCE
+//#endif
 
 /**
  * Copyright (c) 2006-2020 Arseny Kapoulkine
